scripts/upgrade-integration.sh

```diff
@@ -98,7 +97,7 @@ echo "${contents}" > $DAEMON_HOME/config/genesis.json
 
 # Add state data here if required
 
-cosmovisor start >> sifnode.log 2>&1  &
+cosmovisor start --home ~/.sifnoded/ --p2p.laddr 0.0.0.0:27655  --grpc.address 0.0.0.0:9096 --grpc-web.address 0.0.0.0:9093 --address tcp://0.0.0.0:27659 --rpc.laddr tcp://127.0.0.1:26657 >> sifnode.log 2>&1  &
```

Comment:
same here

Reply:
Again, I don't think anyone is using this script ?